### PR TITLE
scroll bar

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
scroll bar

### CODE REVIEW

File: stories/Atom/Navigation/Scroll-bar/ScrollBar.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./scrollbar.scss";
+
+export const ScrollBar = ({}) => (
+  <>
+    <div className="Scrollbar-horizontal-wrapper Scrollbar-horizontal">
+      <div class="Scrollbar-data">.</div>
+    </div>
+    <div className="Scrollbar-vertical-wrapper Scrollbar-vertical">
+      <div class="Scrollbar-data__verticle"></div>
```
Comment (surbhi90) [inline stories/Atom/Navigation/Scroll-bar/ScrollBar.jsx:10]: @harishkumar-srijan Please correct the vertical spelling

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comment

File: stories/Atom/Navigation/Scroll-bar/ScrollBar.jsx (hunk 1 above)
Comment (harishkumar-srijan) [reply stories/Atom/Navigation/Scroll-bar/ScrollBar.jsx]: Done

**Review by surbhi90 — APPROVED** [review]

File: stories/Atom/Navigation/Scroll-bar/ScrollBar.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { Meta, Story } from '@storybook/addon-docs';
+import { ScrollBar } from './ScrollBar';
+
+<Meta title="Atoms/Navigation/ScrollBar"/>
```
Comment (surbhi90) [inline stories/Atom/Navigation/Scroll-bar/ScrollBar.stories.mdx:4]: @harishkumar-srijan Please correct the placement of the component as per the tree structure

**Pull request merged**